.travis.yml

```diff
@@ -4,7 +4,7 @@ services:
 language: go
 sudo: required
 go:
-- 1.7
+- 1.7.x
```

Comment:
Is it OK for you to have different version of golang in Travis and in Integration Tests?
We should rather have the same version everywhere, so we should pick just one from:
a) Latest minor wildcard 
b) Specific version

Currently, we have both :(